runtime/lua/vim/lsp/_inlay_hint.lua

```diff
@@ -206,10 +206,12 @@ end
 ---@param bufnr (integer) Buffer handle, or 0 for current
 ---@private
 function M.disable(bufnr)
-  bufnr = resolve_bufnr(bufnr)
-  clear(bufnr)
-  bufstates[bufnr].enabled = nil
-  bufstates[bufnr].timer = nil
+  if bufstates[bufnr] and bufstates[bufnr].enabled then
+    bufnr = resolve_bufnr(bufnr)
```

Comment:
Shouldn't this still resolve the bufnr before accessing `bufstates` ? Otherwise it could access `bufstates[0]` which is probably not what it is supposed to do.

Reply:
Fixed, thanks